jenkins/jobs/macros-common.yaml

```diff
@@ -152,6 +152,24 @@
           pip install -r swift-uploader/requirements.txt
           swift-uploader/swiftuploader/upload.py -r DFW --password $PASSWD -c CILogs {upload_source} $LOG_PATH
 
+- builder:
+    name: osci-swift-upload-logs-with-console-vgpu
+    builders:
+      #- shell: "/usr/local/jenkins/slave_scripts/grab_console_log.sh"
+      - shell: |
+          #!/bin/sh
+          ls -l test.log logs
+          BUILD_TAG=${BUILD_TAG:-"temp_build"}
+          LOG_PATH=${LOG_PATH:-$BUILD_TAG}
+          OPENSTACK_LOGS=/tmp/opentack_logs
```

Comment:
What uses this variable? Just the cp below?

Reply:
It is a hard code path which would be used in test_vgpu.sh. I use the same name for easy recognition.

Reply:
Will change it